scenarioo-client/protractor-e2e-scenarioo.conf.js

```diff
@@ -30,16 +31,17 @@ var exportsConfig = {
     directConnect: true,
 
     // Timeouts: https://angular.github.io/protractor/#/timeouts
-    allScriptsTimeout: 20000,
-    getPageTimeout: 20000,
+    allScriptsTimeout: 5000,
+    getPageTimeout: 5000,
 
     capabilities: {
         browserName: 'chrome',
         chromeOptions: {
             args: [
-                'disable-infobars',
-                'window-size=1280,800',
-                'headless'
+                '--disable-infobars',
+                '--window-size=1280,800',
+                '--headless',
+                '--disable-gpu'
```

Comment:
nötig? Warum? Auch ok für lokale ausführung?

Reply:
Just following the protractor docs: https://github.com/angular/protractor/blob/master/docs/browser-setup.md#using-headless-chrome